<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioWidgetManager</class>
 <widget class="QWidget" name="AudioWidgetManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblAudioMiddleware">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Audio Middleware:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="cmbAudioMiddleware">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnScanAudio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Audio Banks:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
  <action name="actionAddCategory">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
   </property>
   <property name="text">
    <string>Add Category</string>
   </property>
   <property name="toolTip">
    <string>Adds a new category to group sounds</string>
   </property>
  </action>
  <action name="actionRenameCategory">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-rename.png</normaloff>:/icons16x16/generic-rename.png</iconset>
   </property>
   <property name="text">
    <string>Rename Category</string>
   </property>
   <property name="toolTip">
    <string>Renames the selected category</string>
   </property>
  </action>
  <action name="actionRemoveCategory">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove Category</string>
   </property>
   <property name="toolTip">
    <string>Removes the selected category</string>
   </property>
  </action>
  <action name="actionAddAudioBank">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
   </property>
   <property name="text">
    <string>Add New Audio Bank</string>
   </property>
   <property name="toolTip">
    <string>Create a new Audio Bank</string>
   </property>
  </action>
  <action name="actionDeleteAudioBank">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Selected Audio Bank</string>
   </property>
   <property name="toolTip">
    <string>Deletes the selected Audio Bank</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/HyEditor.qrc"/>
 </resources>
 <connections/>
</ui>
